pkg/data/management/role_data.go

```diff
@@ -52,7 +52,7 @@ func addRoles(wrangler *wrangler.Context, management *config.ManagementContext)
 		addRule().apiGroups("").resources("secrets").verbs("create").
 		addRule().apiGroups("management.cattle.io").resources("cisconfigs").verbs("get", "list", "watch").
 		addRule().apiGroups("management.cattle.io").resources("cisbenchmarkversions").verbs("get", "list", "watch").
-		addRule().apiGroups("rke-machine-config.cattle.io").resources("*").verbs("create").
+		addRule().apiGroups("rke-machine-config.cattle.io").resources("*").verbs("create", "get").
```

Comment:
Standard users shouldn't be able to get all machine configs. They should only be able to get the machine configs they create or specifically given access to.

There is a controller in Rancher that gives standard users access to machine configs they specifically create. My guess is that the terraform provider is moving on before this controller has time to grant this access.

Reply:
Ok that makes sense. I was wondering why standard users couldn't GET machine configs.. it's possible. If you run the apply twice, then terraform allows a standard user to create the cluster. Fix here may be to have the provider wait until that access is granted